d2-schemas/src/main/pegasus/com/linkedin/d2/D2RelativeStrategyProperties.pdl

```diff
@@ -0,0 +1,125 @@
+namespace com.linkedin.d2
+
+/**
+ * Relative load balancer strategy properties for a particular D2 service.
+ *
+ * Balances traffic to hosts within a service by dynamically adjusting a server's health score
+ * based on call statitics compared relatively to the performance of the entire cluster.
+ *
+ * Health score is rated on a scale from 0.0 - 1.0, with 0.0 meaning most unhealthy (all traffic
+ * routed away) and 1.0 meaning most healthy (no traffic routed away). Note that this behavior is
+ * inverse of dropRate in the degrader strategy.
+ */
+record D2RelativeStrategyProperties {
```

Comment:
Is the PDL change independent from the refactoring? If we can remove PDL change in this PR, we should be able to commit this change first.

Reply:
Refactoring depends on PDL change for a small part, specifically the DelegatingRingFactory.